bin/setup-wordpress.sh

```diff
@@ -25,12 +20,26 @@ if [[ ! -e "$LANDO_MOUNT/public/core-dev/.svn" ]]; then
     rm -rf tmp-svn
 fi
 
-if [[ ! -e "$LANDO_MOUNT/public/core-dev/wp-config.php" ]]; then
-    cp "$LANDO_MOUNT/config/wp-config.php" "$LANDO_MOUNT/public/core-dev/wp-config.php"
+CONFIG_FILE="$LANDO_MOUNT/public/core-dev/wp-config.php"
```

Comment:
I don't really like writing a PHP file from here, looks quite messy. I think we should stick to having actual files in `config` directory to copy.

Reply:
Changed now to use symlink.